### DISCUSSION TITLE
SSH port forward tunnel to SOC

### DISCUSSION THREAD

**Discussion by rmarkscott** [Security-Onion-Solutions/securityonion#3645] [Unsupported Versions]
SOC is running on 192.168.170.29. I want to ssh tunnel into the network and access it with port forwarding. Here is the command line I am using:

ssh -L 443:192.168.170.29:443 64.22.x.x 

It connects to the SSH server but with this message.

bind [127.0.0.1]:443: Permission denied
channel_setup_fwd_listener_tcpip: cannot listen to port: 443
Could not request local forwarding.

Can anyone offer any guidance? SOC is accessible within the network.

**Comment by launchdaemon** [comment]
You can't forward port 443 on your local host without sudo as that is a privileged port.  You would need to do something like:

`ssh -L 9443:192.168.170.29:443 64.22.x.x`

and then enter https://localhost:9443 in your browser.

**Comment by rmarkscott** [comment]
Thanks for your response, but I tired that as well and it gives the same problem.

**Reply by launchdaemon** [reply]
Are you sure you don't already have something listening on port 443?  Ubuntu actually tells me I need root if I try a similar command to you but only get the same error when there is something already listening rather than just not having privileges to listen on it.

**Comment by TOoSmOotH** [comment]
The main issue that prevents this is the strict cookie checking. You will need to change your url_base to something that matches what you do on the tunnel. This will break all non-tunnel users.
